app/guide/using-markdown.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Using Markdown
+description: A guide to using Markdown within design history posts
+eleventyNavigation:
+  parent: Guide
+---
+
+Posts within this design history website are written using [Markdown](https://en.wikipedia.org/wiki/Markdown), a simple markup language which saves you from having to use HTML.
+
+## Paragraphs
+
+You don’t have to do anything special for paragraphs, except for making sure that there is an empty line between each one.
+
+```md
+This is a paragraph.
+
+This is another paragraph.
+This sentence will appear next to the previous sentence as single line breaks are ignored.
+```
+
+## Headings and sub-headings
+
+For headings, start the line with hash (`#`) characters, followed by a space.
+
+You should use 2 hash characters for a heading (h2), or 3 hash characters for a sub-heading (h3).
+
+Do not include a post title (h1) as this will be generated automatically from the information at the top of the post.
+
+You do not need to create a heading before the introductory paragraph.
+
+```md
+This is an introductory paragraph.
+
+## Heading
+
+This is a paragraph.
+
+### Sub-heading
+
+This is a paragraph.
+```
+
+## Links
+
+To create links, you have to use a combination of square brackets and round brackets.
+
+The square brackets come first, and contain the link text. This must be immediately followed by round brackets which contain the URL.
+
+```md
+For more information, see [vaccinations on the NHS website](https://www.nhs.uk/vaccinations/).
+```
+
+### Internal links
+
+If you are linking to another post on this design history site, you should not include the full URL, and instead just the part after `https://design-history.prevention-services.nhs.uk`.
+
+This should start with a forward slash (`/`) character:
+
+```md
+We previously wrote about [the pilot](/screening-invite/2025/06/invite-pilot-overview/).
+```
+
+## Lists
+
+You can add bullet lists by starting each item of the list with a dash (`-`) character followed by a space. You can also use asterisks (`*`).
+
+There should be a blank line before and after each list.
+
+```md
+We have 3 user types:
+
+- administrators
+- clinicians
+- managers
+```
+
+### Numbered lists
+
+If the order of items in your list is significant, use a numbered list.
+
+Each item in the list should start with a number, a full stop and then a space.
+
+```md
+This was our design process:
+
+1. desk research
+2. explore lots of different designs
+3. refine a couple of options
+4. usability testing
```

Comment:
```suggestion
1. Desk research
2. Explore lots of different designs
3. Refine a couple of options
4. Usability testing
```

I think the [style guide uses](https://service-manual.nhs.uk/design-system/styles/typography#numbered-lists) capital letters at the start of ordered lists.

Reply:
Ah good point!